routes/videos.router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+const router = express.Router();
+const { Video } = require('../models/video.model');
+
+router.get('/', async (req, res) => {
+    try {
+        const videos = await Video.find({});
```

Comment:
videos variable is null if there are no videos.. So it's better to send an empty list rather than null. like you could say res.json({ success: true, videos || [] });

Reply:
`find()` by default returns `[]` if nothing is found. `findOne()` and `findById()` return null if nothing is found.